Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -146,11 +150,18 @@ cannot be guaranteed to be immutable.
 The listeners
 -------------
 
-Extensions and PHP packages can add listeners that are registered via YAML. They
+Extensions and PHP packages can add listeners that are registered via YAML (or starting
+with TYPO3 v13 also natively in :ref:`PHP using attributes <EventDispatcherImplementation>`). They
 are usually associated to :php:`Event` objects by the fully-qualified class name
 of the event to be listened on. It is the task of the listener provider to
 provide configuration mechanisms to represent this relationship.
 
+If multiple event listeners for a specific event are registered, their order can
+be configured or an event listener can also be overridden with a different one.
+
+The TYPO3 `Configuration` backend module reveals an easy overview of all registered
```

Comment:
We use the guilabel role when mentioning modules, also the full path:

    :guilabel:`System > Configuration`

Additionally, please mention that the lowlevel system extension must be active for the Configuration module to be available. Additionally, no need to write "TYPO3 backend module", just "backend module" is sufficient.

NB: Please check if the main module is really System.

Reply:
Thanks! Yes, Configuration is in System. Addressed both things, very good. I'll try to remember this.